native/src/main/java/io/ballerina/stdlib/websubhub/NativeHubService.java

```diff
@@ -57,7 +57,12 @@ public Set<String> getRemoteMethodNames() {
         return methodParameterMapping.keySet();
     }
 
-    public List<Type> getMethodParameters(String methodName) {
-        return methodParameterMapping.getOrDefault(methodName, Collections.emptyList());
+    public Object[] resolveArgs(String methodName, InteropArgs interopArgs) {
+        List<Type> argType = methodParameterMapping.getOrDefault(methodName, Collections.emptyList());
```

Comment:
Should we have variable name like argTypeList?

Reply:
What abt `argTypes` ?